### ISSUE TITLE
Version 1.6.4:  Webpack Build Broken with message "NodePath has been removed so is read-only"

### ISSUE THREAD

**Issue by J9th** [ant-design/babel-plugin-import#209]
**Issue:**
Update to 1.6.4 breaks the webpack build: "webpack --config webpack.config.js"

**Env:**
Occurs in Windows / Linux
Node Version: v7.9.0 and v9.5.0 (haven't tried in other versions)

```
"babel-polyfill": "^6.26.0",
"babel-core": "^6.26.0",
"babel-loader": "^7.1.2",
"babel-plugin-import": "^1.4.0",
"babel-plugin-transform-class-properties": "^6.24.1",
"babel-plugin-transform-decorators-legacy": "^1.3.4",
"babel-preset-env": "^1.6.1",
"babel-preset-react": "^6.24.1",
"webpack": "^3.10.0",
```

**Workaround:**
Revert back to version 1.6.3 (build works fine in the previous version).
Latest module version breaks something, as comparing to the previous build only this node module was updated.

**File In Question:**
app/main.js

```
import React from 'react';
import ReactDOM from 'react-dom';
import App from './components/App';
import enUS from 'antd/lib/locale-provider/en_US';
import { LocaleProvider } from 'antd';
import { useStrict } from 'mobx';
useStrict(true);

ReactDOM.render(<LocaleProvider locale={enUS}><App /></LocaleProvider>, document.getElementById('app'));
```

**Error:**
``` ERROR in ./app/main.js
Module build failed: SyntaxError: /ad_jenkins/workspace/mr_build/app/main.js: NodePath has been removed so is read-only. (Thian internal node. Probably an internal error)
    at File.buildCodeFrameError (/ad_jenkins/workspace/mr_build/node_modules/babel-core/lib/transformation/file/index.js:427:
    at NodePath.buildCodeFrameError (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/index.js:140:26)
    at NodePath._assertUnremoved (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/removal.js:70:16)
    at NodePath.remove (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/removal.js:20:8)
    at /ad_jenkins/workspace/mr_build/node_modules/babel-plugin-import/lib/Plugin.js:122:18
    at Array.forEach (<anonymous>)
    at Plugin.ProgramExit (/ad_jenkins/workspace/mr_build/node_modules/babel-plugin-import/lib/Plugin.js:121:26)
    at applyInstance (/ad_jenkins/workspace/mr_build/node_modules/babel-plugin-import/lib/index.js:34:27)
    at PluginPass.exit (/ad_jenkins/workspace/mr_build/node_modules/babel-plugin-import/lib/index.js:81:7)
    at newFn (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/path/context.js:117:8)
    at TraversalContext.visitQueue (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/context.js:192:19)
    at Function.traverse.node (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/index.js:114:17)
    at traverse (/ad_jenkins/workspace/mr_build/node_modules/babel-traverse/lib/index.js:79:12)
    at File.transform (/ad_jenkins/workspace/mr_build/node_modules/babel-core/lib/transformation/file/index.js:548:35)
    at /ad_jenkins/workspace/mr_build/node_modules/babel-core/lib/transformation/pipeline.js:50:19
    at File.wrap (/ad_jenkins/workspace/mr_build/node_modules/babel-core/lib/transformation/file/index.js:564:16)
    at Pipeline.transform (/ad_jenkins/workspace/mr_build/node_modules/babel-core/lib/transformation/pipeline.js:47:17)
    at transpile (/ad_jenkins/workspace/mr_build/node_modules/babel-loader/lib/index.js:50:20)
    at Object.module.exports (/ad_jenkins/workspace/mr_build/node_modules/babel-loader/lib/index.js:175:20)
 @ multi babel-polyfill ./app/main.js
```

Let me know if you need more info

**Comment by ThePainnn** [comment]
+1 ... man lost 2 hours figuring this out, don't know why this page has not appear in my search!

1.6.3 seem to work fine.

**Comment by J9th** [comment]
Yeah I spent a couple of hours trying to figure this out as well. Looked at my last 2 previous jenkins build (one successful and one failed) and saw the only difference was that this module was updated and then tried a revert. Thankfully it worked so I could continue my work.

**Comment by christian-acuna** [comment]
same here.

**Comment by yesmeck** [comment]
Can anyone provide a reproduction repo?

**Comment by wengwengweng** [comment]
@yesmeck https://github.com/wengwengweng/cosmetic-web you can use this to Repair problem.

**Comment by yesmeck** [comment]
Should be fixed in 1.6.5
